Documentation/tutorial/intro_rest_api_documentation.md

```diff
@@ -0,0 +1,103 @@
+Introduction to REST API documentation
+===============================
+
+Introduction
+-------------
+
+DocFX now supports generating documentation from REST APIs following [Swagger specification](http://swagger.io/specification/) version 2.0.
+
+The Swagger RESTful API files *MUST* be end with `.swagger.json` or `.swagger2.json` or `_swagger.json` or `_swagger2.json` so that these files are recognized as REST API files in DocFX.
```

Comment:
"." is not required for `_swagger2.json` and `_swagger.json`. Actually `contacts_swagger2.json` is aad's naming convention.